src/components/ec-navigation/types.ts

```diff
@@ -0,0 +1,11 @@
+export interface NavigationBranding {
+  logo: string,
+  name: string,
+}
+
+export interface NavigationProps {
+  isCollapsed: boolean,
+  isCollapsable: boolean,
+  branding: NavigationBranding,
```

Comment:
is this prop required?

Reply:
I made it required. JS version didn't require it but the implementation requires it. It's not a big change, EBO passes the branding prop all the time.